examples/huggingface/README.md

```diff
@@ -0,0 +1,145 @@
+# Implementation of NLP algorithms using Huggingface & Zenml
+
+These examples demonstrate how we can use zenml and huggingface transformers to build, train, & test NLP models.
+
+Huggingface; one of our favorite emoji to express thankfulness, love, or appreciation. In the world of AI/ML, [`Hugging Face`](https://huggingface.co/) is a startup in the Natural Language Processing (NLP) domain (now they are expanding to computer vision and RL) , offering its library of SOTA models in particular around the Transformers. More than a thousand companies using their library in production including Bing, Apple, Microsoft etc. Do checkout thier [`Transformers Library`](https://github.com/huggingface/transformers), [`Datasets Library`](https://github.com/huggingface/datasets) and [`Model Hub`](https://huggingface.co/models)
+
+NLP is a branch of machine learning that is about helping systems to understand natural text and spoken words in the same way that humans do.
+
+The following is a list of common NLP tasks:
+
+- Classification of sentences: sequence-classification
+- Classification of each words in a sentence: token-classification
+- Extraction of answer from a context text: question-answering
+- Text generation using prompt: text-generation
+- Translation: text-translation
+
+## Sequence Classification
+
+Sequence Classification is an NLP/NLU task, where we assign labels to a given text, i.e. sentiment classification, natural langauge inference etc. In this example, we will train a sentiment classification model using the [`imdb`](https://huggingface.co/datasets/imdb) dataset.
+
+- Load dataset: Load sequence-classification dataset in this case it is the `imdb` dataset
+```python
+    from datasets import load_dataset
+    datasets = load_dataset("imdb")
+    print(datasets['train'][0])
+
+    {'label': 0, # Sentiment label i.e. 0->Negative 1->Positive
+ 'text': 'I rented I AM CURIOUS-YELLOW from my video store because of all the controversy that surrounded it when it was first released in 1967. I also heard that at first it was seized by U.S. customs if it ever tried to enter this country, therefore being a fan of films considered "controversial" I really had to see this for myself.......'}
```

Comment:
Could you remove the indent here and add some manual linebreak to the text as it renders in a long scroll?